app/helpers/trends_helper.rb

```diff
@@ -27,7 +27,7 @@ def trend_title(trend)
     end
   end
 
-  def chart_data(trend, from = Setting.max_trend, to = Time.zone.now)
+  def chart_data(trend, from = Setting.max_trend, to = Time.now.utc)
```

Comment:
Did you have to make this change? I ask because `Time.zone` should have a timezone and according to the [Rails documentation](http://api.rubyonrails.org/v4.2.4/classes/Time.html#method-c-zone), it may not be UTC.